Functions.Templates/Templates/BlobTrigger-CSharp-Isolated/BlobTriggerCSharp.cs

```diff
@@ -14,13 +14,12 @@ public BlobTriggerCSharp(ILogger<BlobTriggerCSharp> logger)
             _logger = logger;
         }
 
-        [Function(nameof(BlobClientFunction))]
+        [Function(nameof(BlobStreamFunction))]
```

Comment:
Should match the name of the class

```suggestion
        [Function(nameof(BlobTriggerCSharp))]
```